packages/vision/src/helpers.js

```diff
@@ -196,6 +210,9 @@ module.exports = apiVersion => {
    *
    *   If the key is `content`, the value should be a Buffer.
    * @param {Object=} options
+   *   Any additional options which should be mapped onto the
```

Comment:
I think we need to remove the `image` documentation above, and perhaps rename `options` to `request`.

Then, the docs for `request` can be:

```js
* @param {Object} request
*   A representation of the request being sent to the Vision API.
*   This is an [AnnotateImageRequest]{@link AnnotateImageRequest}.
* @param {Object} request.image
*   A dictionary-like object representing the image. This should have a
*   single key (`source`, `content`).
*
*   If the key is `source`, the value should be another object containing
*   `imageUri` or `filename` as a key and a string as a value.
*
*   If the key is `content`, the value should be a Buffer.
```

I believe the docs are incorrect in places where `request` is marked as optional. `Object=` notation means optional, and should just be `Object` if it is required.